.github/workflows/ubuntu-workflow.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with: 
+          submodules: true
```

Comment:
I think we should move away completely from using submodules if possible.

Reply:
I totally agree, we have a [task](https://github.com/Flank/flank/issues/896) for it, but for now, this is a critical issue and should be fixed fast